app/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -362,20 +360,20 @@ msgstr ""
 
 #: app/templates/error.html:6
 msgid "Sorry, something went wrong"
-msgstr "PLACEHOLDER WELSH Sorry, something went wrong"
+msgstr "Mae’n ddrwg gennym, aeth rhywbeth o’i le"
 
 #: app/templates/error.html:9
 msgid "We are experiencing some technical problems."
-msgstr "PLACEHOLDER WELSH We are experiencing some technical problems."
+msgstr "Rydym yn profi rhai problemau technegol"
 
 #: app/templates/error.html:12
 #, python-format
 msgid ""
 "Please try again later, or %(open)scontact us%(close)s if the problem "
 "continues."
 msgstr ""
-"PLACEHOLDER WELSH Please try again later, or %(open)scontact us%(close)s "
-"if the problem continues."
+"Ceisiwch eto nes ymlaen, neu cysylltwch â ni os bydd y broblem yn parhau."
+" %(open)scontact us%(close)s \" //TODO contact u"
```

Comment:
A quick look on translate suggests that `contact us` in welsh is  `cysylltwch â ni`, it looks like we already have that as the `contact us` link in one other place too so I think it's safe to assume that's what it should be here.